docs/faq.rst

```diff
@@ -291,6 +292,86 @@ See :ref:`encrypted_repos` for more details.
           user
           <https://security.stackexchange.com/questions/14000/environment-variable-accessibility-in-linux/14009#14009>`_.
 
+Using ``BORG_PASSCOMMAND`` with a properly permissioned file
+  Another option is to create a file with a password in it in your home
+  directory and use permissions to keep anyone else from reading it. For
+  example, first create a key::
+
+    head -c 1024 /dev/urandom | xxd -p > ~/.borg-passphrase
```

Comment:
`xxd` should be converting it to hex here. I thought (mistakenly) that it was in coreutils and I could assume people would have it installed.

As for why I was using 30 chars, it was the boundary for a line break in `xxd`, and I was averse to having newlines in the password. Of course, I could have fixed it with a simple `tr`, but that would have complicated the instructions. In hindsight, I don't see why I was so dead-set on not having newlines in the password anyway...

I'll change that to use `base64`, which is *actually* in GNU coreutils (and seems to exist on Mac, too).